test/test_automatic_segmentation.py

```diff
@@ -65,46 +65,63 @@ def tearDown(self):
             import torch.mps
             torch.mps.empty_cache()
 
+    # get the predictor and segmenter.
+    def _get_predictor_and_segmenter(self, ais=False, is_tiled=False, amg_kwargs={}):
```

Comment:
I think it would make sense to refactor this to `automatic_instance_segmentation.py` as a proper documented function to make it easy for users to load the predictor and segmenter.